esp-wifi-sys/src/lib.rs

```diff
@@ -12,3 +14,70 @@ pub mod c_types;
 #[cfg_attr(feature = "esp32s2", path = "include/esp32s2.rs")]
 #[cfg_attr(feature = "esp32s3", path = "include/esp32s3.rs")]
 pub mod include;
+
+#[cfg(feature = "sys-logs")]
+pub mod log {
+    #[no_mangle]
+    pub unsafe extern "C" fn rtc_printf(s: *const u8, args: ...) {
+        syslog(0, s, args);
+    }
+
+    #[no_mangle]
+    pub unsafe extern "C" fn phy_printf(s: *const u8, args: ...) {
+        syslog(0, s, args);
+    }
+
+    #[no_mangle]
+    pub unsafe extern "C" fn coexist_printf(s: *const u8, args: ...) {
+        syslog(0, s, args);
+    }
+
+    #[no_mangle]
+    pub unsafe extern "C" fn net80211_printf(s: *const u8, args: ...) {
+        syslog(0, s, args);
+    }
+
+    #[no_mangle]
+    pub unsafe extern "C" fn pp_printf(s: *const u8, args: ...) {
+        syslog(0, s, args);
+    }
+
+    #[no_mangle]
+    pub unsafe extern "C" fn syslog(_priority: u32, _format: *const u8, _args: ...) {
```

Comment:
this needs to be

```rust
pub unsafe extern "C" fn syslog(_priority: u32, _format: *const u8, _args: core::ffi::VaListImpl) {
```

to make fix the crashes

Reply:
I thought they were the same thing :shrug: - thanks, I was completely lost!